module/CMakeLists.txt

```diff
@@ -86,6 +86,10 @@ list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/traffic_cop")
 list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/transport")
 list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/voltage_domain")
 
+if (SCP_MODULE_EXCLUDE_PATHS)
```

Comment:
IIUC, SCP_MODULE_EXCLUDE_PATHS will be defined in Firmware.cmake ? in that case can you add some comment
e.g.

```
# Remove paths if this variable is defined by the platform target's Firmware.cmake
if (SCP_MODULE_EXCLUDE_PATHS)
```

Reply:
Correct, SCP_MODULE_EXCLUDE_PATHS is defined in Firmware.cmake. Will update patch to add comment.